src/jvmMain/kotlin/com/sdercolin/vlabeler/ui/editor/labeler/marker/Marker.kt

```diff
@@ -619,6 +629,15 @@ private fun MarkerState.handleCursorPress(
             val forcedDrag = action == MouseClickAction.MoveParameterIgnoringConstraints
             cursorState.update { startDragging(lockedDrag, withPreview, forcedDrag) }
         }
+        if (appConf.editor.clickToSwitchCurrentIndex) {
+            cursorStateValue.position?.let { position ->
+                getEntryIndexByCursorPosition(position)?.let { index ->
+                    // having bugs here
+                    // if the cursor do not move, it would not jump
```

Comment:
Do you mean, when only clicking and without any moving, this line is not executed?
It doesn't happen on my side.
Could you explain more?